poker/utils/action.py

```diff
@@ -0,0 +1,32 @@
+from poker.utils.chip import GameStack
+
+
+class Action:
+
+    CHECK = "check"
```

Comment:
What about grouping them in an enum like you did with FaceCards?

Reply:
I'll go that route. I wasn't certain about making class attributes to begin with.